lib/templates.js

```diff
@@ -571,6 +571,47 @@ function _getPackageID(namespace, packageIds, resourceType, visibility = RESOURC
     return packageIds.find((id) => id.includes(`-${resourceType}-`)) || packageIds.find((id) => id.includes(namespace));
 }
 
+/**
+ * This is a template function to create MCP API Resource object.
+ *
+ * @param {object} appConfig - The application configuration.
+ * @param {Array} packageIds - The available package identifiers.
+ * @param {Array} accessStrategies The array of accessStrategies objects
+ * @returns {Object} An MCP API resource object.
+ */
+const createMCPAPIResourceTemplate = (appConfig, packageIds, accessStrategies) => {
+    const ordId = `${appConfig.ordNamespace}:apiResource:mcp-server:v1`;
+    const packageId = _getPackageID(
+        appConfig.ordNamespace,
+        packageIds,
+        ORD_RESOURCE_TYPE.api,
+        RESOURCE_VISIBILITY.public,
+    );
+
+    return {
+        ordId,
+        title: `MCP Server for ${appConfig.appName}`,
+        shortDescription: `This is the MCP server to interact with the ${appConfig.appName}`,
+        description: `This is the MCP server to interact with the ${appConfig.appName}`,
+        version: "1.0.0",
+        lastUpdate: appConfig.lastUpdate,
+        visibility: RESOURCE_VISIBILITY.public,
+        partOfPackage: packageId,
+        releaseStatus: "active",
+        apiProtocol: "mcp",
+        resourceDefinitions: [
+            {
+                type: "custom",
+                mediaType: "application/json",
+                url: `/ord/v1/${ordId}/mcp-server-definition.mcp.json`,
+                accessStrategies,
+            },
+        ],
+        entryPoints: [],
+        extensible: { supported: "no" },
```

Comment:
@Fannon , here is the draft, please comment, how you would envision the MCP server to be described